app/scripts/context/layer-data.tsx

```diff
@@ -51,9 +51,13 @@ const fetchLayerById = async (
       }
     };
   } else {
-    const domain = data.summaries
+    let domain = data.summaries
       ? data.summaries.datetime
       : data.extent.temporal.interval[0];
+    const lastDatetime = domain[domain.length - 1]
+    if (lastDatetime == null) {
+      domain[domain.length - 1] = new Date().toISOString()
```

Comment:
I don't think I understand what is going on with temporal interval. Can you explain why this step is necessary for netCDF data?

Reply:
For each of these colletions, the CMR STAC record's second temporal extent is null (see [GPM_3IMERGDF.v07](https://cmr.earthdata.nasa.gov/stac/GES_DISC/collections/GPM_3IMERGDF.v07) and [TRMM_3B42_Daily.v7](https://cmr.earthdata.nasa.gov/cloudstac/GES_DISC/collections/TRMM_3B42_Daily.v7),